webapp/packages/core-app/src/TopNavBar/ConnectionSchemaManager/ConnectionSelector/ConnectionSelectorController.ts

```diff
@@ -143,14 +145,19 @@ export class ConnectionSelectorController {
   }
 
   private getConnectionItems(): IMenuItem[] {
-    return Array.from(this.connectionInfo.data.values()).map(item => {
-      const menuItem: IMenuItem = {
-        id: item.id,
-        title: item.name || item.id,
-        onClick: () => this.connectionSelectorService.selectConnection(item.id),
-      };
-      return menuItem;
-    });
+    return Array.from(this.connectionInfo.data.values())
```

Comment:
this.connectionInfo.values instead `Array.from(this.connectionInfo.data.values())`